CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 - Support sending to Bitcoin taproot addresses
+- Fix opening 'transactions export' CSV file
+- Added Dutch translation
```

Comment:
"Add", imperative? reads nicer when everything is in the same mood. then also, Migrated -> Migrate